File: clientside/src/app/entities/notifikacija.ts
export class Notifikacija {
    id:number;
    tip:String;
    tekst:String;
    vreme:Date;
    procitana:boolean;
    ikona:String;
    link:String;
    constructor(tip:String,tekst:String,vreme:Date,link?:String)
    {
        this.tip=tip;
        this.tekst=tekst;
        this.vreme=vreme;
        this.procitana=false;
        switch(tip){
            case "error":
                this.ikona="error";
                break;
            case "warning":
                this.ikona="warning";
                break;
            case "success":
                this.ikona="check_outline";
                break;
            case "info":
                this.ikona="info";
                break;
            default:
                break;
        }
        this.link=link;
    }
}

File: Core/SmartGrid/Models/Notifikacija.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace SmartGrid.Models
{
    public class Notifikacija
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public string Tip { get; set; }
        [Required]
        public string Tekst { get; set; }
        [Required]
        public DateTime Vreme { get; set; }
        [Required]
        public bool Procitana { get; set; }
        [Required]
        public string Ikona { get; set; }

        public string Link { get; set; }
    }
}
